akmod-specs/displaylink.spec

```diff
@@ -0,0 +1,138 @@
+Name:		displaylink
+Version:	5.4.0
+%global config_release 1
+Release:	1%{?dist}
+Summary:	Meta-package for proprietary DisplayLinkManager application
+URL:		https://www.synaptics.com/products/displaylink-graphics/downloads/ubuntu
+Source0:	https://www.synaptics.com/sites/default/files/exe_files/2021-04/DisplayLink%20USB%20Graphics%20Software%20for%20Ubuntu5.4-EXE.zip
+Source1:	https://github.com/displaylink-rpm/displaylink-rpm/archive/v%{version}.tar.gz
+License:	MIT
+Requires:	akmod-evdi, %{name}-config, %{name}-manager
+
+%description
+Meta-package to install kernel module and proprietary tools
+needed to communicate with and manage DisplayLink USB graphics
+adapters along with additional config files.
+
+%files
+
+%build
+
+%global _hardened_build 1
+
+%package manager
+Summary:	Proprietary DisplayLink Manager application
+License:	Redistributable, no modification permitted
+URL:		https://www.synaptics.com/products/displaylink-graphics/downloads/ubuntu
+ExclusiveArch:	armv7hl i386 x86_64
+BuildRequires:	libdrm-devel
+# The DisplayLinkManager binary is linked at run-time to a specific version of libevdi
+Requires:	libevdi == 1.9.1
+
+%description manager
+This contains the proprietary tools needed to communicate with and manage
+DisplayLink USB graphics adapters.
+
+%files manager
+%license LICENSE
+%dir %{_libexecdir}/%{name}
+%{_libexecdir}/%{name}/DisplayLinkManager
+%{_libexecdir}/%{name}/ella-dock-release.spkg
+%{_libexecdir}/%{name}/firefly-monitor-release.spkg
+%{_libexecdir}/%{name}/ridge-dock-release.spkg
+%dir /var/log/%{name}/
+%{_prefix}/lib/systemd/system-sleep/displaylink.sh
+
+%package config
+Summary:	Supporting config files for DisplayLink USB graphics adapters
+License:	MIT
+URL:		https://github.com/displaylink-rpm/displaylink-rpm
+BuildArch:	noarch
+Requires:	%{name}-manager logrotate
+
+%description config
+This adds support for HDMI/VGA adapters built upon the DisplayLink DL-6xxx,
+DL-5xxx, DL-41xx and DL-3xxx series of chip-sets. This includes numerous
+docking stations, USB monitors, and USB adapters.
+
+%preun config
+%systemd_preun displaylink.service
+
+%postun config
+%systemd_postun_with_restart displaylink.service
+
+%files config
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
+%{_unitdir}/displaylink.service
+%{_udevrulesdir}/99-displaylink.rules
+%{_datadir}/X11/xorg.conf.d/20-displaylink.conf
+%{_presetdir}/95-displaylink.preset
+
+%post config
+%systemd_post displaylink-service
```

Comment:
Should this be a dot instead? `displaylink.service` instead of `displaylink-service`?